Documentation/Sources/getting-started.md

```diff
@@ -1,8 +1,8 @@
-#  Getting started with AEPNotificationContent extension
+#  Installing the AEPNotificationContent package
 
 ### Prerequisites
 
-Make sure you have completed the steps in the [prerequisites](./prerequisites.md)
+Make sure you have completed the steps in the [prerequisites](./../README.md#prerequisites).
```

Comment:
One minor update to line no 19 in this file
Can we change `type` to `run` ?
Replace `YOUR_NOTIFICATION_CONTENT_EXTENSION_NAME` and then, in the `Podfile` directory, type:
to 
Replace `YOUR_NOTIFICATION_CONTENT_EXTENSION_NAME` and then, in the `Podfile` directory, `run`: